src/Mvc/View/Engine/Twig.php

```diff
@@ -16,19 +16,24 @@ class Twig extends Engine implements EngineInterface
     /**
      * @var \Twig_Environment
      */
-    protected $twig;
+    protected $_twig;
```

Comment:
Why you change variable? unnecessary.

Reply:
Actually is a Phalcon convention, even if is not explicit in the documentation, protected and private variables are prefixed with a underscore, just look at the variable of Phalcon core classes.

Reply:
Let me know with some link, please? We are using the PSR2 convention. But, if you are right, we follow :)

Reply:
@gabriel-schneider

Reply:
The Phalcon [Application class](https://github.com/phalcon/cphalcon/blob/master/phalcon/application.zep) is a example of this.

Reply:
@gabriel-schneider Ohh man, this is zephir. Different way. Core system, older. If you see, [Bootstrap.php](https://github.com/phalcon/phalcon-devtools/blob/master/scripts/Phalcon/Bootstrap.php), wrote totally in php don't used underscore. Zephir don't have $, by this uses underscore as easy way to visibility indentify.